assets/queries/k8s/workload_host_port_not_specified/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "id": "2b1836f1-dcce-416e-8e16-da8c71920633",
-  "queryName": "Image Host Port Not Specified",
+  "queryName": "Workload Host Port Not Specified",
   "severity": "LOW",
   "category": "Networking and Firewall",
-  "descriptionText": "Sees if Kubernetes Image Host Port is Specified",
+  "descriptionText": "Verifies if Kubernetes workload's host port is specified",
   "descriptionUrl": "https://kubernetes.io/docs/concepts/containers/images/#updating-images",
```

Comment:
https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/#exposing-the-service